### PR TITLE
update: Okta steps

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## Describe your changes
- Adds instructions for configuring relay state
- Adds troubleshooting for invalid relay state errors

## Checklist

- [ ] The first paragraph of the page is on one line.
- [ ] The other lines have a line break at 90 characters.
- [X] I checked the output.
- [ ] I applied the [style guide](../styleguide.md).
- [ ] My links start with `/docs/`.

### CODE REVIEW

**Review by acutaia — APPROVED** [review]
LGTM!

File: docs/platform/howto/saml/add-okta-idp.md [hunk 1]
```diff
@@ -66,6 +70,13 @@ Login failed. Please contact your account administrator for more details.
 
 Ensure **IdP initiated login** is enabled.
 
+### Invalid relay state
+
+If you get the invalid relay state error, then you are attempting an
```
Comment (wojcik-dorota) [inline docs/platform/howto/saml/add-okta-idp.md:75]: You might remove "then" (wordiness)

File: docs/platform/howto/saml/add-okta-idp.md [hunk 2]
```diff
@@ -66,6 +70,13 @@ Login failed. Please contact your account administrator for more details.
...
+IdP-initiated auth flow. This happens, for example, when you click the
+Aiven SAML app in Okta. Set the **Default Relay State** in Okta to the
+Aiven console that your organization uses.
```
Comment (wojcik-dorota) [inline docs/platform/howto/saml/add-okta-idp.md:78]: You might want to capitalize "console".

**Review by wojcik-dorota — APPROVED** [review]
Looks good! Two minor suggestions.

File: docs/platform/howto/saml/add-okta-idp.md (hunk 2 above)
Comment (staceysalamon-aiven) [reply docs/platform/howto/saml/add-okta-idp.md:78]: It's the general noun here, not the proper noun (there are multiple consoles).

**Pull request merged**